README.md

```diff
@@ -53,8 +55,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+The likely memory complexity O(1), since the only memory needed to traverse a path is for one pointer or two - one on each end - to move through the path.
```

Comment:
# -3

At the very least you have to store the input so you can do the comparison, so you're looking at O(n).